### PR TITLE
SHS-5608: Collection: Title Settings field updates

### PR DESCRIPTION
**Pull request by hectorlj**
# READY FOR REVIEW

## Summary
Changed the following for `hs_collections` component:

- Added helper text to `Title Settings` field `See the User Guide for help on how to use this option`
- Moved the Title Settings  field above the Title  field on the Collection paragraph type form display
- Changed the `Title Settings` option `The Collection title should be visible as a Heading` to `The Collection title should be visible as a Heading 2`

## Steps to Test
1. Edit/create a flexible page
2. Add a Collection component
3. The changes outlined above are visible in the Collection paragraph type form display

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

### CODE REVIEW

File: config/default/field.field.paragraph.hs_collection.field_title_settings.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ field_name: field_title_settings
 entity_type: paragraph
 bundle: hs_collection
 label: 'Title Settings'
-description: ''
+description: 'See the <a href="https://hsweb.slite.page/p/mc94Sg92YTiY7V/Collection-Guide#23102d46">User Guide</a> for help on how to use this option.'
```
Comment (cienvaras) [inline config/default/field.field.paragraph.hs_collection.field_title_settings.yml]: Please add a `target="_blank"` to the link to have it open in a new tab.

**Review by cienvaras — CHANGES_REQUESTED** [review]
@hectorlj Works great, thanks. Just added a comment requesting a minor change to improve usability.

File: config/default/field.field.paragraph.hs_collection.field_title_settings.yml (hunk 1 above)
Comment (hectorlj) [reply config/default/field.field.paragraph.hs_collection.field_title_settings.yml]: Fixed!

**Review by cienvaras — APPROVED** [review]
@hectorlj Thanks! 🚀

**Pull request merged**